lib/connection_manager.go

```diff
@@ -35,24 +33,10 @@ type ConnectionManager struct {
 	// doesn't need a reference to the Server object. But for now we keep things lazy.
 	srv *Server
 
-	// When --connectips is set, we don't connect to anything from the addrmgr.
-	connectIps []string
-
-	// The address manager keeps track of peer addresses we're aware of. When
-	// we need to connect to a new outbound peer, it chooses one of the addresses
-	// it's aware of at random and provides it to us.
-	AddrMgr *addrmgr.AddrManager
 	// The interfaces we listen on for new incoming connections.
 	listeners []net.Listener
 	// The parameters we are initialized with.
 	params *DeSoParams
-	// The target number of outbound peers we want to have.
-	targetOutboundPeers uint32
-	// The maximum number of inbound peers we allow.
-	maxInboundPeers uint32
-	// When true, only one connection per IP is allowed. Prevents eclipse attacks
-	// among other things.
-	limitOneInboundConnectionPerIP bool
```

Comment:
Same thing here. Revert deletion of these fields here to reduce scope of code that's just moved around

Reply:
Similarly to my comment about `connectIps`, these fields will only be used by the connection_controller. I think makes more sense now to keep them there.